bridge/bridgetest/bridgetest.go

```diff
@@ -20,7 +20,7 @@ func readPbFrame(conn net.Conn) (data []byte, err error) {
 	var len uint32
 	err = binary.Read(conn, binary.LittleEndian, &len)
 	if err != nil {
-		return
+		return nil, err
```

Comment:
Note that these and others below make use of [named return values](https://go.dev/tour/basics/7), so these changes have no effect.